wrapper/src/main/java/software/amazon/jdbc/dialect/DialectManager.java

```diff
@@ -66,7 +66,7 @@ public class DialectManager implements DialectProvider {
         }
       };
 
-  protected static final long ENDPOINT_CACHE_EXPIRATION = TimeUnit.MINUTES.toNanos(30);
+  protected static final long ENDPOINT_CACHE_EXPIRATION = TimeUnit.HOURS.toNanos(24);
```

Comment:
NIT: let's add a comment to explain what this value represents
"this value indicates how long... etc etc"